src/Integration.Vsix/Commands/HelpMenu/AboutDialog.xaml

```diff
@@ -7,37 +7,42 @@
     SizeToContent="WidthAndHeight"
     Background="WhiteSmoke">
     <Window.Resources>
+        <Style TargetType="{x:Type Hyperlink}">
+            <Setter Property="FocusVisualStyle" Value="{x:Null}"/>
+            <Setter Property="Foreground" Value="Blue" />
+            <Setter Property="TextBlock.TextDecorations" Value="{x:Null}" />
+            <Setter Property="TextOptions.TextFormattingMode" Value="Ideal"/>
+            <Setter Property="TextOptions.TextRenderingMode" Value="ClearType"/>
+            <Setter Property="FontFamily" Value="..\..\Resources\MavenPro-Regular.ttf #Maven Pro"/>
+            <Style.Triggers>
+                <Trigger Property="IsMouseOver" Value="True">
+                    <Setter Property="Foreground" Value="Red" />
+                </Trigger>
+            </Style.Triggers>
+        </Style>
         <Style x:Key="TextBlockStyle" TargetType="TextBlock">
-            <Setter Property="FontSize" Value="12"/>
             <Setter Property="Foreground" Value="Black"/>
+            <Setter Property="TextOptions.TextFormattingMode" Value="Ideal"/>
+            <Setter Property="TextOptions.TextRenderingMode" Value="ClearType"/>
+            <Setter Property="FontFamily" Value="..\..\Resources\MavenPro-Regular.ttf #Maven Pro"/>
         </Style>
     </Window.Resources>
-    <Grid Margin="8,8,8,8">
+    <Grid>
```

Comment:
Having a grid that is essentially a 1x1 square defeats the purpose of the grid. It's doable of course, but it's harder to read and maintain when elements have such large margins. It is also more error-prone as it's difficult to see when elements are not aligned properly, something that is better achieved via grid rows and columns. For example, the last row (copyright and close button) should be on the alignment, but because they are standalone elements, it is harder to do.
![image](https://user-images.githubusercontent.com/60586879/207610326-aa2e587b-fda2-4889-a5d0-abaf2da2e7af.png)

I think there should be a 2x2 grid:
* Row 0, Column 0: picture and version
* Row 0, Column 1: version
* Row 1, Column 0: copyright
* Row 1, Column 1: button

With appropriate margins and alignments. WDYT?